cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/ReplicatorBuilder.java

```diff
@@ -44,6 +44,14 @@
      */
     public static class Push extends ReplicatorBuilder<Datastore, URI, Push> {
 
+        private int changeLimitPerBatch = 500;
+
+        private int batchLimitPerRun = 100;
+
+        private int bulkInsertSize = 10;
+
```

Comment:
I meant the default settings could be stored somewhere and used to set values consistently throughout since it appeared that the values did repeat in the code.  I'm fine with your explanation, if you feel strongly against doing it then I defer to your judgement.